crates/sc-consensus-subspace/src/archiver.rs

```diff
@@ -627,7 +627,7 @@ where
                     ))
                 })?;
             // We need to limit number of threads to avoid running out of WASM instances
-            let blocks_to_archive = thread_pool.install(|| {
+            let blocks_to_archive: Vec<(SignedBlock<_>, _)> = thread_pool.install(|| {
```

Comment:
We already specify the type in `.collect()` below, wouldn't it be better to specify the fuller type there or what is the issue with IDE in this case specifically?